radixsort.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+import math
+import random
+import time
+
+
+def timed_func(func):
+    """Decorator for timing our insertion sort methods."""
+    def timed(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        elapsed = time.time() - start
+        print "time expired: %s" % elapsed
+        return result
+    return timed
+
+
+@timed_func
+def radixsort(l):
+    """Implemented Radix Sort using math notation"""
```

Comment:
docstrings should describe the action the function will take, not the action you took to create the function.